sample/build.gradle

```diff
@@ -18,14 +18,13 @@ android {
     }
 }
 
-ext.supportLibraryVersion = '27.1.0'
-
 dependencies {
-    implementation "com.android.support:appcompat-v7:${supportLibraryVersion}"
-    implementation "com.android.support:support-v4:${supportLibraryVersion}"
+    implementation 'androidx.appcompat:appcompat:1.1.0-alpha02'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'com.google.android.material:material:1.1.0-alpha03'
```

Comment:
Required`com.google.android.material:material` for resolve dependencie with `androidx.legacy:legacy-support-v4`.